template-formats/prevalance-visualisation-format/years1980-2026/scenario1/Countries.csv

```diff
@@ -0,0 +1,4 @@
+country_code, country_measure, age_start, age_end, mean, 2.5_percentile, 5_percentile, 10_percentile, 25_percentile, 50_percentile, 75_percentile, 90_percentile, 95_percentile, 97.5_percentile, standard_deviation, median
+NGA, prevalance, 1, 9, 0.2, 0.1, 0.1, 0.1, 0.1, 0.2, 0.3, 0.3, 0.3, 0.3, 0.1, 0.2
+NGA, passed_prevalance_test, NA, NA, 
```

Comment:
Very, very, very tiny nit: could we add NAs to match the total number of columns in the csv?